CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# 2020-07-13
+
+## Release Notes
+
+* If you are self-hosting wire on the public internet, consider [changing your brig server config](https://github.com/wireapp/wire-server/blob/49f414add470f4c5e969814a37bc851e26f6d9a7/docs/reference/user/registration.md#blocking-creation-of-personal-users-new-teams-refrestrictregistration).
+* Deploy all services except nginz.
+* No migrations, no restrictions on deployment order.
+
+## New Features
+
+* Implement active flag in SCIM for user suspension (#1158)
+
+## Bug Fixes
+
+* Restrict user creation in on-prem installations (#1161)
```

Comment:
I'd argue that this is a feature. What do you think?

Reply:
I disagree both with the "please squash the changelog commits" and the categorization of #1161, but fine.  :)

Reply:
Oh, okay. Well, I'm happy to approve now, but I'd be happy to discuss this. Would be nice to understand your perspective.